code-of-conduct.md

```diff
@@ -1,3 +1,37 @@
 # OpenTelemetry Community Code of Conduct
 
-OpenTelemetry follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
\ No newline at end of file
+The OpenTelemetry project aims to be a welcoming place where new and existing
+members feel safe to respectfully share their opinions and disagreements. We
+want to attract a diverse group of people to collaborate with us, which means
+acknowledging that people come from different backgrounds and cultures. 
+
+We are bound by the [CNCF Code of
+Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) and
+covered by that in more serious violations and prefer the reporting of
+violations to be made to us as outlined below.
+
+## Reporting violations
+
+If you have seen or experienced unacceptable behavior or anything that would
+make our community less welcoming, please speak up! 
+
+The Governance Committee is the right place to bring your concerns and can be
+reached via cncf-opentelemetry-governance@lists.cncf.io. You can also get in
+touch with any committee member if you feel safer this way: any one of us can
+receive your report and will bring it to the entire group for investigation,
```

Comment:
I don't think all reports should always be brought to the entire group and I wouldn't promise that here. It may prevent someone from raising a complaint against a GC member.

Reply:
Agreed, I replaced "will" by "may":

> any one of us can receive your report and may bring it to the entire group for investigation